Modules/Core/Common/src/itkFloatingPointExceptions_unix_feenableexcept_using_fegetenv.cxx

```diff
@@ -87,11 +87,16 @@ itk_fedisableexcept(const fexcept_t excepts)
 
 // Implementation for Intel
 
-#  if (defined ITK_HAS_STRUCT_FENV_T_CONTROL)
+// Note: on Apple, ignore the result of the try_compiles that generate these
```

Comment:
`try_compile` was a must 20 years ago (when ITK was largely designed) due to very varied compiler standards adherence. It is problematic for cross-compiling. And it makes configure step longer. Reducing our dependence on it is welcome.

Reply:
may give that a shot one day, but for now this small change is probably safer, especailly with 5.2 so close.